website/versioned_docs/version-2.0.6/contribute/contribution-guidelines.md

```diff
@@ -54,9 +54,9 @@ This will build the project by downloading dependencies as Go modules.
 
 #### Running Go tests
 
-All code we check into our repo needs to have sufficient tests to ensure it is maintainable and works as expected. You can use Go run tests in Prysm. 
+All code we check into our repo needs to have sufficient tests to ensure it is maintainable and works as expected.
 
-Many tests are rely on the Bazel build system, thus testing with `go test` would not work.
+Many tests rely on the Bazel build system, thus testing with `go test` will not work.
```

Comment:
```suggestion
Many tests rely on the Bazel build system, thus testing with `go test` may not work.
```

I would say that most of the tests should work with `go test`, but bazel is preferred for many of the more complicated build requirements and test setup.